openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/EntityRightCollapsablePanel.spec.ts

```diff
@@ -75,10 +71,6 @@ test('Show and Hide Right Collapsable Panel', async ({ page }) => {
     page.locator('[data-testid="KnowledgePanel.GlossaryTerms"]')
   ).toBeVisible();
 
-  await expect(
-    page.locator('[data-testid="KnowledgePanel.CustomProperties"]')
-  ).toBeVisible();
-
```

Comment:
why we are removing checks?

Reply:
As it's a conditional widget, and only visible if there is any custom property available for the Entity. Since we are only checking collapsable panel behaviour here. We can rely on other static widget!